src/chart/animator/keyframe.cpp

```diff
@@ -25,6 +25,11 @@ void Keyframe::init(const Gen::PlotPtr &plot)
 			emptyOpt->reset();
 			if (source && source->getOptions()->title.get())
 				emptyOpt->title = source->getOptions()->title.get();
+			if (source && source->getOptions()->subtitle.get())
```

Comment:
checking source in each if is redundant

Reply:
OK

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/349/commits/9b47b8203c365ef3df340538926f33efe992da9f